src/basic/missing_syscall.h

```diff
@@ -134,7 +134,7 @@ static inline int missing_memfd_create(const char *name, unsigned int flags) {
 #  endif
 #elif defined(__powerpc__)
 #  define systemd_NR_getrandom 359
-#elif defined(__s390x__)
+#elif defined(__s390__)
```

Comment:
you are saying `__s390__` is always set if `__s390x__`, did I get this right?

Reply:
Right.